src/test/functional/features/final-order.feature

```diff
@@ -66,14 +68,22 @@ Feature: Final order
     And I set the case state to "FinalOrderOverdue"
     When I click "Sign out"
     And I login with applicant "2"
-
-    Given I go to "/"
     Then the page should include "Your wife has still not applied for a 'final order'"
-    Given I click "Apply for a final order"
-    Then the page URL should be '/respondent/finalising-your-application'
+    And the page URL should be '/respondent/hub-page'
 
-    Given I select "I want permission to apply for a final order, and to finalise my divorce"
+    Given a superuser updates "dateFinalOrderEligibleToRespondent" with "2020-01-01"
+    When I set the case state to "AwaitingFinalOrder"
+    And I click "Apply for a final order"
```

Comment:
Might be best to keep the page URL check after this step
```suggestion
    And I click "Apply for a final order"
    Then the page URL should be '/respondent/finalising-your-application'
```

Reply:
Changed as Lara suggested